Constants.js

```diff
@@ -0,0 +1,5 @@
+module.exports = Object.freeze({
```

Comment:
Nice compartmentalization of constants in this constants file + usage of `Object.freeze`